tests/pr_test.go

```diff
@@ -2,44 +2,63 @@
 package test
 
 import (
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/common"
+	"log"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
 const completeExampleDir = "examples/complete"
+const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-resources.yaml"
+
+var permanentResources map[string]interface{}
+
+// TestMain will be run before any parallel tests, used to read data from yaml for use with tests
+func TestMain(m *testing.M) {
+	var err error
+	permanentResources, err = common.LoadMapFromYaml(yamlLocation)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Exit(m.Run())
+}
 
 func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  dir,
-		Prefix:        prefix,
-		ResourceGroup: resourceGroup,
+		Testing:      t,
+		TerraformDir: dir,
+		Prefix:       prefix,
+		Region:       "us-south", // Locking to "us-south" as the permanent KMS instance is located in us-south
```

Comment:
Can you remove this? I know we have it everywhere else, but its not true. You can use KSM from any region (and even any account). However just check what regions appid is supported in, as you may need to do something like [this](https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/blob/4aa0eef337b0643c59f82dab184d7716520b1338/tests/pr_test.go#L26-L32) to ensure valid regions are selected